test/runBTagAnalyzer_cfg.py

```diff
@@ -715,9 +716,9 @@
 #-------------------------------------
 
 #-------------------------------------
-if options.runOnData:
+#if options.runOnData:
     ## Remove MC matching when running over data
-    removeMCMatching( process, ['All'] )
+    #removeMCMatching( process, ['All'] )
```

Comment:
Have you checked that the cfg file now works correctly with real data? There are several MC-specific modules in the PAT workflow that need to be disables when running over real data.